understanding/20/focus-order.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Focus Order</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Keyboard users navigate content in a correct order</dd>
-		<dt>Author task</dt><dd>Elements receive focus in an order that preserves meaning</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Keyboard users navigate content in a correct order.</dd>
+         <dt>What to do</dt><dd>Elements receive focus in an order that preserves meaning.</dd>
+         <dt>Why it's important</dt><dd>Navigating a site or application with only the keyboard will make sense.</dd> 
```

Comment:
@rainbreaw commented:
> In understanding/20/bypass-blocks.html, understanding/20/focus-order.html, understanding/20/focus-visible.html and understanding/20/keyboard-no-exception.html:

> Consider extending why it's important to include individuals who use switch devices or other AT inputs.

Reply:
I'm hardly experienced, but to me switches are a slightly different paradigm to keyboard focus order, in that they actually offer a means of bypassing the focus order navigation (linear versus row-column scanning.). I think this SC works better talking about "with only the keyboard". Happy to be told I'm wrong :)

Reply:
Ah, you are correct. In my comment, I was still honing in on the importance of focus being visible even if someone is using a different input... which is not what these SC are about. Let's disregard this comment of mine! Thank you :)